docs/src/examples/components/Menu/Variations/MenuExampleColor.shorthand.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import { Menu, ProviderConsumer, Grid, Text } from '@stardust-ui/react'
+
+const items = [
+  { key: 'editorials', content: 'Editorials' },
+  { key: 'review', content: 'Reviews' },
+  { key: 'events', content: 'Upcoming Events' },
+]
+
+const iconItems = [
+  { key: 'home', content: 'Home', icon: 'home' },
+  { key: 'users', content: 'Users', icon: 'users' },
+  { key: 'search', icon: 'search' },
+]
+
+const MenuExampleColor = () => (
+  <ProviderConsumer
+    render={({ siteVariables: { colorScheme } }) => {
+      const colorsArr = _.keys(colorScheme)
+      const colors = _.times(7, num => colorsArr[num % colorsArr.length])
+
+      return (
+        <Grid
+          columns="repeat(2, auto)"
+          styles={{ justifyContent: 'left', justifyItems: 'left', alignItems: 'center' }}
+          variables={{ gridGap: '10px' }}
+        >
+          <Text content={`${_.upperCase(colors[0])} DEFAULT MENU:`} weight="bold" />
+          <Menu defaultActiveIndex={0} color={colors[0]} items={items} />
+          <Text content={`${_.upperCase(colors[1])} PILLS MENU:`} weight="bold" />
+          <Menu defaultActiveIndex={0} color={colors[1]} pills items={items} />
+          <Text content={`${_.upperCase(colors[2])} POINTING MENU:`} weight="bold" />
+          <Menu defaultActiveIndex={0} color={colors[2]} pointing items={items} />
+          <Text content={`${_.upperCase(colors[3])} VERTICAL POINTING MENU:`} weight="bold" />
+          <Menu defaultActiveIndex={0} color={colors[3]} vertical pointing items={items} />
+          <Text content={`${_.upperCase(colors[4])} UNDERLINED MENU:`} weight="bold" />
+          <Menu defaultActiveIndex={0} color={colors[4]} underlined items={items} />
+          <Text content={`${_.upperCase(colors[5])} ICON MENU:`} weight="bold" />
+          <Menu defaultActiveIndex={0} color={colors[5]} items={iconItems} />
+          <Text content={`${_.upperCase(colors[6])} ICON ONLY MENU:`} weight="bold" />
+          <Menu
+            defaultActiveIndex={0}
+            color={colors[6]}
+            iconOnly
+            items={iconItems.map(item => _.pick(item, ['key', 'icon']))}
+          />
+          <Text content={`UNDERLINED MENU (mutiple colors):`} weight="bold" />
+          <Menu
+            defaultActiveIndex={0}
+            underlined
+            color={{ foreground: colors[4], background: colors[5], border: colors[1] }}
+            items={items}
+          />
+        </Grid>
+      )
+    }}
+  />
+)
+
+export default MenuExampleColor
```

Comment:
for tracking @layershifter 's comment in: https://github.com/stardust-ui/react/pull/712#discussion_r247543678

> This example looks overcomplicated to me, may be we can:
> 
> * introduce a new `Select` knob
> * use it to get select an active color

will address

Reply:
@levithomason 
in SUIR there are not that many versions of Menus; I just introduced a Select Knob at @layershifter 's proposal and now we have examples that look like this:

TODO - introduce screenshot

We can separate these into different example files where we reuse the knob or we can add a knob on every example for the color, but because of the way knobs are implemented we need to have a `.knobs.tsx` file for every example, which is a lot of duplication

what do you think?